openfisca_uk/parameters/demographic/population_estimate.yaml

```diff
@@ -0,0 +1,37 @@
+description: Population estimates for UK nations
+ENGLAND:
+  metadata:
```

Comment:
I'm guessing we need `metadata` for each nation for it to work with the interpolation. If this is correct could you add a comment noting as such?

Reply:
Yep, added